assets/src/blocks/Spreadsheet/SpreadsheetEditor.js

```diff
@@ -0,0 +1,107 @@
+import {useState} from '@wordpress/element';
+import {InspectorControls} from '@wordpress/block-editor';
+import ColorPaletteControl from '../../block-editor/ColorPaletteControl/ColorPaletteControl';
+import {SpreadsheetFrontend} from './SpreadsheetFrontend';
+import {debounce} from '@wordpress/compose';
+import {TextControl, PanelBody} from '@wordpress/components';
+
+const {__} = wp.i18n;
+
+const colors = [
+  {name: 'blue', color: '#167f82'},
```

Comment:
We should validate these colors with the Design Team. I suspect most of them were removed from our palette

Reply:
Hey @GP-Dan-Tovbein these are colors found in the token [file](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/base/_tokens.scss#L49).

Reply:
These are the correct colors, we updated them when working on the new identity

Reply:
Gotcha! But shouldn't be better, and consistent, if we respect the same names as it is in the tokens file? I mean
```
blue -> blue-green
green -> p4-dark-green-800
grey -> grey-800
gp-green -> gp-green-800 
```

Reply:
I see your point, but as I said in this [comment](https://github.com/greenpeace/planet4-master-theme/pull/2178#discussion_r1439287636) it would require a few changes and also a migration for the existing blocks! If we decide to do it, maybe it should be in a separate PR?

Reply:
We can do something [like this](https://github.com/greenpeace/planet4-master-theme/blob/main/src/Migrations/M026ReplaceDeprecatedColorsFromContent.php#L19) afterwards, then? 🤔